FEXCore/Source/Interface/Core/OpcodeDispatcher/AVX_128.cpp

```diff
@@ -450,7 +455,6 @@ OpDispatchBuilder::RefPair OpDispatchBuilder::AVX128_LoadSource_WithOpSize(
     AddressMode HighA = A;
     HighA.Offset += 16;
 
-    ///< TODO: Implement VSIB once we get there.
     if (Operand.IsSIB()) {
       const bool IsVSIB = (Op->Flags & X86Tables::DecodeFlags::FLAG_VSIB_BYTE) != 0;
       LOGMAN_THROW_AA_FMT(!IsVSIB, "VSIB currently unsupported");
```

Comment:
assert msg is stale, change to something like "VSIB uses LoadVSIB instead"

Reply:
Changed